### PR TITLE
hamburger thingy shows left bar by default. Press hamburger to see right...

### PR DESCRIPTION
**Pull request by ahsu1230**
... column.

Basically just flipped from the original function

@andrewconner @yipingliao @stkem @2is10

### CODE REVIEW

**Comment by andrewconner** [comment]
Product knows about it? May want to double check. I like the idea.

**Comment by ahsu1230** [comment]
Yup, will double check.

**Comment by yipingliao** [comment]
Awesome!

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl [hunk 1]
```diff
@@ -17,14 +17,14 @@
 
 @media only screen and (max-width: 1024px)
   .kf-sidebar
-    left: -135px
-    z-index: -1
+    left: 0
+    z-index: 1
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:21]: Why does the z-index need to be lowered from `10` to `1` on this screen size?

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl [hunk 2]
```diff
@@ -17,14 +17,14 @@
 
 @media only screen and (max-width: 1024px)
   .kf-sidebar
-    left: -135px
-    z-index: -1
+    left: 0
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:20]: What does this `left: 0` accomplish? If nothing, let’s omit it.
Oh, is it required to make the transition work?

Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:20]: it makes the transition much smoother. Without it, it gets pretty choppy

File: kifi-backend/modules/shoebox/angular/src/layout/header/HeaderCtrl.js [hunk 3]
```diff
@@ -9,13 +9,14 @@ angular.module('kifi')
...
-      $rootElement.find('html').toggleClass('kf-sidebar-active');
+      $rootElement.find('html').toggleClass('kf-sidebar-inactive');
     };
 
+    // reminder: this triggers left sidebar to show in case the default for small windows is to hide left sidebar
     $window.addEventListener('message', function (event) {
       if (event.data === 'show_left_column') {  // for guide
         $scope.$apply(function () {
-          $rootElement.find('html').addClass('kf-sidebar-active');
+          $rootElement.find('html').addClass('kf-sidebar-inactive');
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/header/HeaderCtrl.js]: Scratch my earlier comment. Shouldn’t this be `.removeClass` now?

**Comment by 2is10** [comment]
nice

File: kifi-backend/modules/shoebox/angular/src/layout/header/HeaderCtrl.js (hunk 3 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/layout/header/HeaderCtrl.js]: if we removed the class, wouldn't we have to `.addClass` later? 
Oh wait. nvm.

~ edit 1 ~ if we `.removeClass` then would we still be able to `.toggleClass` afterwards?

**Comment by ahsu1230** [comment]
@weeewake says it's a go!

**Pull request merged**